### DISCUSSION TITLE
프로젝트 관리(Project Management)

### DISCUSSION THREAD

**Discussion by myrhymetree** [sixes-sense/back-end#44] [Announcements]
# 프로젝트 관리
프로젝트 관리는 팀이 프로젝트에 있는 업무를 정리, 추적, 수행하도록 도와줍니다. <br>
여기서 말하는 프로젝트란 특정 목표를 달성하기 위한 일련의 작업을 의미합니다. <br>
프로젝트 관리를 통해 팀이 업무를 계획, 관리, 수행하여 적시에 프로젝트 요구 사항을 충족할 수 있습니다. <br>
[프로젝트관리 참고] (https://asana.com/ko/resources/benefits-project-management)

## 마일스톤
프로젝트 마일스톤은 프로젝트 타임라인에서 특정 지점을 나타냅니다. 이는 단일 또는 그룹으로 된 활동이 완료된 때나 새로운 단계나 활동이 시작되는 때를 나타내는 체크포인트입니다. 

마일스톤은 프로젝트 계획에서 진척 상황을 나타내는 강력한 이정표입니다. 모든 구성원이 마일스톤을 통해 진행 상태를 확인하고 우선순위를 판단할 수 있어 팀을 조율하고 동기를 부여합니다. 또한 마감 일을 모니터링하고, 중요한 날짜를 인지하고, 프로젝트를 지연시킬 가능성이 있는 사항을 발견하는 데 도움이 됩니다. 프로젝트 타임라인에서 작업을 삭제하더라도 마일스톤이 있으면 프로젝트 주요 단계의 개요를 파악할 수 있습니다.

마일스톤에 포함되는 이슈 선택 기준
- 마일스톤과 직접 연관
  백엔드 API  v1.0 마일스톤은 백엔드 API에 필수적(없으면 API가 아닌 정도)으로 필요한 이슈들만 선택했습니다.
  백엔드 v1.0 배포 역시 배포에 선행되는 작업들 위주로 선택했습니다. <br>

 

[마일스톤 참고] (https://asana.com/ko/resources/project-milestones?utm_campaign=NB--KO--KR--Catch-All--All-Device--Broad&utm_source=google&utm_medium=pd_cpc_nb&gclid=CjwKCAjwloynBhBbEiwAGY25dO6jCUQd15pMP8Gs7OtSsRhmEWvvcQcUGIlcP_SwaXcZYgLYab84oBoCojYQAvD_BwE&gclsrc=aw.ds)

## 칸반보드
<img width="950" alt="image" src="https://github.com/sixes-sense/back-end/assets/94158097/ccb21afc-b5f7-4b17-83c9-00835900092e">

Todo에는 해야될 작업(이슈)를 기재하거나 드래그 앤 드랍으로 옮깁니다. <br>
In Progress : 진행 중인 작업(이슈)를 기재하거나 드래그 앤 드랍으로 옮깁니다. <br>
Done : 완료한 작업(이슈)를 기재하거나 드래그 앤 드랍으로 옮깁니다. <br>
** issue 탭에서 이슈를 작성하시면 칸반보드에 자동 연동 됩니다. <br>
[칸반보드 참고] (https://asana.com/ko/uses/kanban-boards?&utm_campaign=&utm_source=&utm_medium=&gclid=CjwKCAjwloynBhBbEiwAGY25dNwCX4xLuIRgJPVfuhaWaz5s1frWgI8lCwhWhICzTdOQeHnOzXJebRoCOtIQAvD_BwE&gclsrc=aw.ds)

## 로드맵(간트차트)
<img width="953" alt="image" src="https://github.com/sixes-sense/back-end/assets/94158097/d8380e1f-ef8b-47b3-9ef2-3631a2f23c88">

1. 로드맵의 좌측 컴포넌트 옆에 '+' 버튼을 클릭합니다.

<img width="640" alt="image" src="https://github.com/sixes-sense/back-end/assets/94158097/9e9fe0ce-0e74-4172-8172-b7284c8ac1c2">

3. 옮겨진 컴포넌트의 '시작일'과 '완료 예정일'을 설정합니다.
** 칸반보드 탭에서 설정한 status는 로드맵과 공유됩니다, 로드맵에서 status 변경해도 칸반보드에서도 적용됩니다.
[간트차트 참고] (https://www.atlassian.com/ko/agile/project-management/gantt-chart)

**왠만하면 칸반이나 간트에 개발 일정을 명확히 작성해주시면 됩니다.(다 같이 상의해야되는 일정이나 예측이 안되는 일정은 일단 날짜는 기재 안하셔도 됩니다, 기간 변경 시 언제든지 변경 가능합니다)**
**누락된 이슈가 있거나 일정 상의 변경이 있다면 원하시는대로 변경해주시면 됩니다.**

**Comment by BearCrying** [comment]
감사합니다!

**Comment by myrhymetree** [comment]
@taegeun-park0525 @kny3037 @bbbbooo @ChoMyeongHwan 
한번 씩 읽어봐주세요 감사합니다.

**Comment by bbbbooo** [comment]
확인했습니다!

**Comment by kny3037** [comment]
확인했습니다! 감사합니다!!
